showcase/src/app/demo-ui/pages/date-picker/demo-date-picker-page.component.html

```diff
@@ -1,47 +1,109 @@
 <h1 class="mat-display-3" translate>SHOWCASE.DEMO.DATE_PICKER.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-picker/reactive-form"
+		exampleTitle="SHOWCASE.DEMO.DATE_PICKER.REACTIVE_FORM"
+	>
+		<mat-form-field>
+			<stark-date-picker
+				pickerId="picker-reactive-form"
+				pickerName="picker-reactive-form"
+				[placeholder]="'SHOWCASE.DEMO.DATE_PICKER.PLACEHOLDER' | translate"
+				[dateFilter]="'OnlyWeekends'"
+				[min]="minDate"
+				[max]="maxDate"
+				dateMask
+				[formControl]="formControl"
+				required
+			></stark-date-picker>
+			<mat-error>
+				<div *ngIf="formControl.hasError('required')">This is required</div>
+				<div *ngIf="formControl.hasError('matDatepickerFilter')">Date filter</div>
+				<div *ngIf="formControl.hasError('matDatepickerMin')">Min date</div>
+				<div *ngIf="formControl.hasError('matDatepickerMax')">Max date</div>
```

Comment:
Can you add some CSS for these errors to show them in one line instead of in separate lines.. otherwise it looks bad:

![datepicker errors styling issue](https://user-images.githubusercontent.com/5698318/54431005-85765e80-4725-11e9-9784-20b24d6fefea.PNG)

Reply:
Maybe could we wait for the integration of `@nationalbankbelgium/ngx-form-errors` in the next release ? 😊